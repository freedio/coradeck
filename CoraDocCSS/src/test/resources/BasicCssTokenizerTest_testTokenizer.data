
"StringToken"
'StringToken'
"OpenString
'OpenString
/* Bad string token */ "
/* Bad string token */ '
#Hashtoken#
#
$=
$
(
)
*=
*
-->
-identifier-like#
-10
-.5
-10.5
-.3e5
-1.5E+12
-0.6E-12
-3e
-10pt
-.5cm
-3.6in
-1.2E6px
-.3in
-5%
-.2%
-99.9%
-
0
1
2
3
4
5
6
7
8
9
12
9999
0123
+10
+.5
+10.5
+.3e5
+1.5E+12
+0.6E-12
+3e
+10pt
+.5cm
+3.6in
+1.2E6px
+.3in
+5%
+.2%
+99.9%
+
,
.
.1
..
.5E+3
.6E-4
.3in
.4E-5km;
/* comment */
/
//
:
:3
;
;;
<!-- CD -->
@
@keyword;
[brack-it]
^=
^
{curlee}
u
u+
u+a
u+ABC?
u+1234567
u+1234???
u+abcd-ef01
u+abc?-ef0?
u+??????
|
|=
||
|||
~
~=
~~=
ident
ident2
I-dent_3
_ident-4
-ident-5
ident-5+ident-6
&%
