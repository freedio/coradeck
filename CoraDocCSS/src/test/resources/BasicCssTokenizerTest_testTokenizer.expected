com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.StringToken
com.coradec.coradoc.token.BadStringToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.StringToken
com.coradec.coradoc.token.BadStringToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.BadStringToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.BadStringToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.BadStringToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.BadStringToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.HashToken
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Matcher
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.ParenthOpen
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.ParenthClose
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Matcher
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.CDC
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Identifier
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerDimension
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerDimension
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealDimension
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealDimension
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealDimension
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealDimension
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerPercentage
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealPercentage
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealPercentage
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerDimension
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerDimension
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealDimension
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealDimension
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealDimension
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealDimension
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.IntegerPercentage
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealPercentage
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealPercentage
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Comma
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealDimension
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.RealDimension
com.coradec.coradoc.token.Semicolon
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Colon
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Colon
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Semicolon
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Semicolon
com.coradec.coradoc.token.Semicolon
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.CDO
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Identifier
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.CDC
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.AtKeyword
com.coradec.coradoc.token.Semicolon
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.BracketOpen
com.coradec.coradoc.token.Identifier
com.coradec.coradoc.token.BracketClose
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Matcher
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.CurlyOpen
com.coradec.coradoc.token.Identifier
com.coradec.coradoc.token.CurlyClose
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Identifier
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Identifier
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.UnicodeRange
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.UnicodeRange
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.UnicodeRange
com.coradec.coradoc.token.IntegerToken
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.UnicodeRange
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.UnicodeRange
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.UnicodeRange
com.coradec.coradoc.token.Identifier
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.UnicodeRange
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Matcher
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Column
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Column
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Matcher
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Matcher
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Identifier
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Identifier
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Identifier
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Identifier
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Identifier
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Identifier
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Identifier
com.coradec.coradoc.token.Whitespace
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.Delimiter
com.coradec.coradoc.token.EOF
